packages/edit-widgets/src/components/widget-areas-block-editor-content/style.scss

```diff
@@ -0,0 +1,3 @@
+.editor-styles-wrapper {
```

Comment:
Minor, but while the `.editor-styles-wrapper` class seems to be needed to pull in particular styles, this classname reuse seems like a bad thing that goes against the BEM-like principles in the project:
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

A trade-off could be to add a second classname:
```html
<div
    className="editor-styles-wrapper edit-widgets-styles-wrapper"
```

and targeting that instead:
```css
.edit-widgets-styles-wrapper {
    position: relative;
}
```
```

Reply:
Good call! Added in 247498e.